beacon_node/beacon_chain/src/beacon_block_streamer.rs

```diff
@@ -864,147 +862,4 @@ mod tests {
             }
         }
     }
-
-    #[tokio::test]
-    async fn check_fallback_altair_to_electra() {
```

Comment:
This test was testing the legacy payload reconstruction, so it's no longer relevant.